templates/with-vercel-website/src/utilities/generatePreviewPath.ts

```diff
@@ -17,10 +17,13 @@ export const generatePreviewPath = ({ collection, slug }: Props) => {
     return null
   }
 
+  // Encode to support slugs with special characters
+  const encodedSlug = encodeURIComponent(slug)
```

Comment:
Why is this being modified in this PR? Pulled in via running the generate script or something?